network/src/commonTest/kotlin/com/wire/kalium/api/v0/call/CallApiV0Test.kt

```diff
@@ -89,6 +89,60 @@ internal class CallApiV0Test : ApiTest() {
         )
     }
 
+    @Test
+    fun givenSftUrlWithHttpProtocol_whenConnecting_thenReplaceItWithHttps() = runTest {
+        val sftConnectionURL = "http://sft.connection.url1/test/endpoint?param1=value1"
```

Comment:
can we also have a test with the SFT URL coming with `HTTPS` protocol? just in case anything changes in the future 😅

Reply:
sure